README.md

```diff
@@ -260,6 +260,36 @@ straightforward (it needs to run with sudo to read environment). Just run:
 | `resticw stats` / `resticw stats snapshot-id ...` | Show the statistics for the whole repo or the specified snapshots |
 | `resticw mount /mnt/restic`                       | Mount your remote repository                                      |
 
+## Setup macOS LaunchAgent
+LaunchAgent is the modern service scheduler in in macOS that uses [Launchd](https://www.launchd.info/).
+[Launchd](https://www.launchd.info/) is the modern built-in service scheduler in macOS. It has support for running services as root (Daemon) or as a normal user (Agent). Here we we set up an LauchAgent to be run as your normal user for starting regular backups.
+
+1. In general, follow the same setup as in (#setup-linux-systemd) except for:
+  * use `make install-launchagent` instead of `make install-systemd`
```

Comment:
_Disclaimer: I've not tried MacOS installation :)_

I was thinking that one of the things I like from root systemd installation is that configuration files, which contains credentials, are quite protected, as they are owned by root and not readable for others.

Any potential malware that runs under normal user permissions will never have access to that credentials, and that gives some sensation of security, specially if you take care about what do you run using `sudo`.

So, I'm not very sure if suggesting the use of `sudo` for MacOS could also bring additional security benefits, i.e. `sudo PREFIX=/usr/local make install-launchagent`. I have no idea if that would work with launchd/launchagent :)

Reply:
I wanted to follow the typical Homebrew setup, which is without root. See https://docs.brew.sh/FAQ#why-does-homebrew-say-sudo-is-bad

Anyway, I'd say if someone has access to your system, that's your bigger problem. Gaining additional access to the backups is probably not going to be much worse than the hacker already having access to the real system. (Given that one uses unique B2 credentials  per bucket, which is recommended, so backups for other systems are not accessed via the token in `pw.txt`)

Reply:
Yep, fine! Just wanted to share my thoughts/curiosity, hehe. Is that I don't like much having readable credential files hanging around :sweat_smile: But I also agree the point of Homebrew about trust and sudo :smile: